.github/workflows/codeql-analysis.yaml

```diff
@@ -0,0 +1,54 @@
+name: "CodeQL"
+
+on:
+  schedule:
+    - cron: '29 10 * * *'
```

Comment:
can you add a human readible comment what schedule this translates to? in pacific time ideally

Reply:
Yeah sure lol this is just some random time it came up with.